modules/ossm-federation-config-import.adoc

```diff
@@ -77,7 +77,7 @@ spec:
   importRules:
   - type: NameSelector
     importAsLocal:
-|Set to `true` to aggregate remote endpoint with local services. When `true`, services will be imported as `<name>.<namespace>.svc.cluster.local`
+|Set to `true` to aggregate remote endpoint with local services. When `true` services will be imported as `<name>.<namespace>.svc.cluster.local`. When 'true' an alias is required. When `false` no alias is required. 
```

Comment:
```
'true'
```
@rh-tokeefe  probably typo. Apostrophes ('') are used instead of Grave accents (``) so that `true` is not highlighted as the others in that section ( https://71852--ocpdocs-pr.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-federation#ossm-federation-config-import_federation )

Reply:
Great catch @mkralik3 ! PR has been updated. I will squash the PR after you review and give it LGTM.